oxo.py

```diff
@@ -1,26 +1,48 @@
+#I don't know whats gone wrong...
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        OxoBoard.board = [[0]*3]*3
```

Comment:
This won't work properly -- it will result in a structure that looks like `[[0,0,0], [0,0,0], [0,0,0]]`, but the inner lists are all references to the same list so changing one of them will change them all (you may recall I mentioned this in class). You need to do something like `[[0]*3 for i in xrange(3)]` to force Python to create multiple lists.

Reply:
Also `OxoBoard.board` will set this up as a variable shared between all instances of the `OxoBoard` class -- doesn't matter if you only have one, but will break the TravisCI script for example. You want to use `self.board` instead.